execFileSystem.go

```diff
@@ -53,9 +60,14 @@ func (fe *FileSystemExecutor) constructTemplate(view *View) (*template.Template,
 			t = out.New(v.Defines)
 		}
 
-		if tree, ok := cache[v.Template]; ok {
-			t.AddParseTree(v.Defines, tree)
-		} else {
+		var (
+			templateString string
+			foundKey       bool
+		)
+		if fe.KeyedString != nil {
+			templateString, foundKey = fe.KeyedString[v.Template]
+		}
+		if !foundKey {
 			buffer.Reset()
 			file, err := fe.FS.Open(v.Template)
```

Comment:
Actually one more thing, is `defer file.Close()` missing here?

Reply:
nice catch, fixed now